app/views/ReportView/components/Settings/components/AddUserDialog/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {
+  useState, useCallback, useContext, useEffect,
+} from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+import api from '@/services/api';
+import UserAutocomplete from '@/components/UserAutocomplete';
+import ReportContext from '@/components/ReportContext';
+import snackbar from '@/services/SnackbarUtils';
+import { UserType } from '@/common';
+
+import './index.scss';
+
+type AddUserDialogProps = {
+  isOpen: boolean;
+  onAdd: (user?: UserType) => void;
```

Comment:
is user unused? `onAdd` doesn't seem to be calling with a user in any of these instances

Reply:
I've removed the param